src/inputs/Switch/Switch.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { PropsWithChildren, useState } from "react";
+
+import * as styles from "./Switch.module.css";
+
+interface SwitchProps {
+  checked?: boolean;
+  disabled?: boolean;
+  onChange(): void;
```

Comment:
**nitpick:** Let's pass value here, so other users could benefit from it. I mean: 
```
onChange(checked: boolean): void;
```